pos_event_sale/hooks.py

```diff
@@ -2,7 +2,37 @@
 # @author Iván Todorovich <ivan.todorovich@gmail.com>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
+import logging
+
 from odoo import SUPERUSER_ID, api
+from odoo.tools.sql import column_exists, create_column
+
+_logger = logging.getLogger(__name__)
```

Comment:
```suggestion
from odoo import SUPERUSER_ID, api
from odoo.tools.sql import column_exists, create_column
```

`_logger` is imported but it doesn't seem to be used